indexer/services/ender/__tests__/helpers/indexer-proto-helpers.ts

```
@@ -451,6 +457,17 @@ export function createOrder({
     };
   }
 
+  if (duration !== undefined && interval !== undefined && priceTolerance !== undefined) {
```

So when TWAP is enabled, all of these fields need to be set right? Should we throw an error if this condition is only partially met?

---

We check these values in stateless validation in message_place_order so think we should be fine here